### PR TITLE
Discount appearence

### PR DESCRIPTION
**Pull request by AlionaN**

### CODE REVIEW

File: src/components/DiscountTag/DiscountTag.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+}) {
+  return (
+    <div className = {styles.discountWrapper}>
+      {percentage === 0 || !percentage
+        ? <div className={`${styles.discount} ${styles.amount}`}>
+          - {flatAmount}$
+          </div>
+        : <div className=
+            {`${styles.discount} ${styles.percent} ${percentage > CENTER_PERCENT ? styles.hot : null}`}
+          >
```
Comment (artemtool) [inline src/components/DiscountTag/DiscountTag.js:17]: Put it in one line.

Comment (artemtool) [reply src/components/DiscountTag/DiscountTag.js:17]: Replace null with "".

File: src/pages/Discounts/components/DiscountList/DiscountList.js [hunk 2]
```diff
@@ -1,12 +1,27 @@
...
+    if (isLike) {
+      favouriteDiscounts.filter((el) => el !== id);
+      setIsLike(false);
+    } else {
+      favouriteDiscounts.push(id);
+      setIsLike(true);
+    }
+  };
+
   const discountsList = discounts?.length
```
Comment (artemtool) [inline src/pages/Discounts/components/DiscountList/DiscountList.js:18]: Do explicit check "discounts?.length > 0".

**Review by artemtool — CHANGES_REQUESTED** [review]
Small changes required.

File: src/components/DiscountTag/DiscountTag.js (hunk 1 above)
Comment (AlionaN) [reply src/components/DiscountTag/DiscountTag.js:17]: I can`t put it in one line, because we have eslint restrictions (120 characters by line)

**Review by artemtool — APPROVED** [review]

**Pull request merged**